<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterwindow</class>
 <widget class="QDialog" name="filterwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="filter_pb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal computation parameters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum depth change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing size</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="depth_sb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cm [0.5-10]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="smooth_sb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>100</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="voxelgrid_cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Voxel Grid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="x_sb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="y_sb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z_sb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cm [1-5]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cm [1-5]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cm [1-5]</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>240</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="bilateral_cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilateral filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma R</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cm [0.1-5]</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>20</width>
     <height>381</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>370</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="median_cb">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Median filter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="windowsize_sb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="maxmov_sb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Window size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cm [1-10]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum movement</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="random_cb">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Random sampling</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="randord_sb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sampling ratio</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="normal_cb">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal sampling</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sampling ratio</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="normord_sb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of bins</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="normbins_sb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="covar_cb">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Covariance sampling</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sampling ratio</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="covarord_sb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[10-100]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[1-20]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[1-20]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[2-10]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[2-10]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[2-10]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[8-32]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
